### ISSUE TITLE
Removes "transform" class

### ISSUE THREAD

**Issue by yogeshgalav** [francoismassart/eslint-plugin-tailwindcss#369]
**Describe the bug**
A linter should never cause a bug, it's the first and only lint plugin I'm regret of using,
It removed transform class from whole of my code base lead to various issue on production,
Now need to add those lines one by one.

**To Reproduce**
Removes "transform" class

**Expected behavior**
Should give error but not autofix things where classes are removed.

**eslint config file or live demo**
```
module.exports = {
    env: {
        browser: true,
        es6: true,
        node: true,
    },
    extends: [
        'eslint:recommended',
        'plugin:vue/vue3-recommended',
        // 'plugin:vue/vue3-essential',
        '@vue/eslint-config-typescript/recommended',
        // 'plugin:tailwindcss/recommended',
        // '@vue/eslint-config-prettier',
        // "airbnb-base"
    ],
    globals: {
        Atomics: 'readonly',
        SharedArrayBuffer: 'readonly',
        ga: true,
    },
    parserOptions: {
        ecmaVersion: 13,
        sourceType: 'module',
        ecmaFeatures: {
            jsx: false,
        },
    },
    plugins: ['vue', 'html'],
    ignorePatterns: ['**/node_modules', '**/vendor'],
    rules: {
        'max-len': ['off'],
        'vue/first-attribute-linebreak': [
            'error',
            {
                singleline: 'ignore',
                multiline: 'below',
            },
        ],
        'vue/max-attributes-per-line': [
            'error',
            {
                singleline: 1,
                multiline: 1,
            },
        ],
        // override/add rules settings here, such as:
        //   'vue/no-unused-vars': 0,
        //   "vue/multi-word-component-names": 0,
        //   "vue/html-self-closing": 0,
        //   "no-unused-vars": 0,
        //   "@typescript-eslint/no-empty-function": 0,
        //   "no-useless-catch": 0,
        //   "@typescript-eslint/no-unused-vars": 0,
        //   "vue/no-unused-components": 0,
        //   "vue/require-explicit-emits": 0,
        //   "@typescript-eslint/no-this-alias": 0,
        //   "no-undef": 0
    },
}

```
